src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -723,5 +725,57 @@ class XAttribute : System.Attribute { }
             Assert.Collection(runResult.TrackedSteps["result_ForAttributeWithMetadataName"],
                 step => Assert.True(step.Outputs.Single().Value is ClassDeclarationSyntax { Identifier.ValueText: "C1" }));
         }
+
+        [Fact]
+        public void ManyBinaryPatterns()
+        {
+            const string Preamble = $"""
```

Comment:
>Preamble [](http://example.com/codeflow?start=25&length=8)

It looks like naming conventions for locals are not followed

Reply:
This what our const convention is.

Reply:
> This what our const convention is. 

I don't think I agree. These are locals and, therefore, shouldn't start with capital letter.

Reply:
There is a bunch of existing constants like that in this file